bcda/auth/client/oktaclient.go

```diff
@@ -229,9 +227,7 @@ func addClientToPolicy(clientID string, requestID uuid.UUID) error {
 	req.Header.Add("Authorization", oktaAuthString)
```

Comment:
This line and the two above can be replaced by your new addRequestHeaders function. There's also two other places where the same replacement can be made. Let's not leave our future selves checking to see why it was done one way in some methods and not in others!